### ISSUE TITLE
Will the model trained with the BTCV dataset work well on other full-body CT images?

### ISSUE THREAD

**Issue by penguin1109** [mindflow-institue/DAEFormer#6]
Hello, thanks for your great work on the multi-organ segmentation task.
I am currently working on the full-body segmentation task as well.
My task is to use PET/CT images and make pseudo labels for the prostate cancer, and to accomplish the task, accurate segmentation maps for the healthy organs are essential.

But of course, the intensity values or sight of view of our CT images are slightly different from the BTCV dataset.
I have tried out the pretrained weights on our dataset,  (I appreciate that you have provided the pretrained checkpoints)
but the <kidney> especially did not seem to segment well on our dataset.

Do you have any recommendations on how to make your pretrained model work well on our dataset, in terms of pre-processing??
Currently, we do not have any ground truth labels, so we only need a rough pseudo label on the organs, but it is important for the kidneys to be correctly segmented.

Also, how did you normalize the intensity values to [0-1]? Was it simply min-max scaling??

I'll be looking forward for your reply.
Thanks.
